app/templates/about_us.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```

Comment:
there is no need to create a full new HTML page for getting the navigation bar. 
Please inherent the navigation:
I'd recommand to copy the "calendar_base.html" file and put your content inside: 
```
    {% block content %}
    {% endblock content %}
   ```

Reply:
I didnt use this instead I added 
`{% extends "base.html" %}`

Reply:
and Ive added the 
`    {% block content %}
    {% endblock content %}`